store/go.mod

```diff
@@ -14,10 +14,10 @@ require (
 	github.com/cosmos/iavl v1.0.0-rc.1
 	github.com/cosmos/ics23/go v0.10.0
 	github.com/linxGnu/grocksdb v1.8.4
+	github.com/mattn/go-sqlite3 v1.14.17
```

Comment:
this is swaping a non cgo dependency for a cgo one,  is that intended?

Reply:
yeah, actually `modernc.org/sqlite` doesn't provide sqlite configuration, I did simple benchmarking and confirmed `go-sqlite3` is faster

Reply:
wanted to double check, thanks for the info 😄